scala/private/common.bzl

```diff
@@ -44,8 +44,8 @@ def collect_plugin_paths(plugins):
             paths.append(p)
         elif hasattr(p, "scala"):
             paths.extend([j.class_jar for j in p.scala.outputs.jars])
-        elif hasattr(p, "java"):
-            paths.extend([j.class_jar for j in p.java.outputs.jars])
+        elif hasattr(p, "JavaInfo"):
```

Comment:
Can we change this to `elif JavaInfo in d`? Similar comment about removing the scala branch.

Reply:
Done.